t/32-openqa_client.t

```diff
@@ -110,8 +112,8 @@ subtest 'upload private assets (local)' => sub {
                 asset => 'private',
                 local => 1
             });
-    };
-    ok !$@, 'No upload errors' or die explain $@;
+    }
+    'No upload errors';
```

Comment:
Why did you remove the `die` here?

Reply:
https://github.com/os-autoinst/openQA/pull/6274